lib/utils/request.js

```diff
@@ -213,14 +215,22 @@ function downloadFile(params, cb){
   var req = requestjs(opts);
 
   req.on('error', function(err){
-    return cb(err);
+    fs.unlink(filePath, function(){
+      return cbOnce(err);
+    });
   });
 
-  req.on('end', function(){
-    return cb();
+  fileStream.on('finish', function(){
+    console.log('Saved to ' + filePath);
```

Comment:
I tried to use the log.info, but it doesn't output anything. The log in fhc is soooo complicated...